app/models/user.rb

```diff
@@ -161,10 +161,10 @@ def tags(limit = 10)
   end
 
   def normal_tags(limit = false)
-    tags(limit).select{ |tag| ! tag.name.include?(':') }
+    tags(limit).select { |tag| !tag.name.include?(':') }
   end
 
-  def tagnames(limit = 20, defaults = true)
+  def tagnames(limit = 20, _defaults = true)
```

Comment:
I'm not sure if this is related to something else, but `_defaults` is not used in the method, why keep it? 😄

Reply:
Done!